golden/runner.py

```
@@ -483,6 +483,201 @@ def _try_l3_dispatch(
+            #    dispatch cost resident weights shrink.
+            if not _bench_enabled():
+                return None
+            try:
+                return _benchmark_l3_resident(
+                    rt, compiled, ordered, run_config, _BENCH_ROUNDS, _BENCH_WARMUP
+                )
+            except Exception as e:  # noqa: BLE001 — benchmark is never a correctness gate
+                print(f"[RUN] benchmark skipped: {type(e).__name__}: {e}", flush=True)
+                return None
+        finally:
+            for name, handle, is_stacked, wid in resident_handles:
```

Ruff (0.15.20)

[warning] Loop control variable `name` not used within loop body

Rename unused `name` to `_name`

(B007)